src/plugins/condition/components/ConditionSet.vue

```diff
@@ -70,6 +70,9 @@ export default {
             this.currentConditionOutput = currentConditionResult.output;
         },
         provideTelemetry() {
+            if (this.stopProvidingTelemetry) {
+                this.stopProvidingTelemetry();
+            }
```

Comment:
@shefalijoshi , Can you explain this to me? I don't think this.stopProvidingTelemetry is ever defined here.

Reply:
It is defined here:
```
this.stopProvidingTelemetry = this.openmct.telemetry
                .subscribe(this.domainObject, output => { this.updateCurrentOutput(output); });
```

Reply:
provideTelemetry() is only called on mounted so this.stopProvidingTelemetry will never be defined at this point.